Add questions.md (#2)
FragileTech/guide: questions.md

# Questions environment setup session

> 1. What are the commands that you are using on the terminal to move across directories?

Reviewer: ```suggestion
1. Can you provide a list of the most useful Terminal commands?
```

> * `cd path` command allows you to navigate to desired path.

Reviewer: Why don't you add a link to a tutorial?
